src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,54 @@
  */
 
 public class Plane {
+    private String type;
+    private int length;
+    private int weight;
+    private int speed;
+    private String brand;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.type = planeBuilder.type;
```

Comment:
```suggestion
        type = planeBuilder.type;
```
і так для всіх полів в конструкторі

Reply:
done!